up42/tasking.py

```diff
@@ -135,10 +139,8 @@ def get_quotations(
             quotation_id (Optional[str], optional): The quotation Id for the specific quotation to retrieve.
             workspace_id (Optional[str], optional): The workspace id (uuid) to filter the search.
             order_id (Optional[str], optional): The order id (uuid) to filter the search.
-            decision (Optional[list[str]], optional): The status of the quotation
-            (NOT_DECIDED, ACCEPTED or REJECTED).
-            sortby (str, optional): Arranges elements in asc or desc order based on a chosen field.
-            The format is <field name>,<asc or desc>.
+            decision (Optional[list[str]], optional): quotation status (NOT_DECIDED, ACCEPTED or REJECTED).
+            sortby (str, optional): Arranges elements in asc or desc order.
```

Comment:
unrelated to this PR:
nitpick: Description seemed a bit conflicting with `descending` param. A personal opinion here :)
```suggestion
            sortby: Param to sort the results. Default value set to createdAt
```